.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 go:
-- 1.6.3
 - 1.7
+- 1.8
```

Comment:
This change is good, but unrelated to this PR. Can you please splice it out into a different branch?